### PR TITLE
git-crecord: init at 20161216.0

### PR DESCRIPTION
**Pull request by matthiasbeyer**
###### Motivation for this change

Would like to test this package and maybe use it myself.

I need help though, as I cannot get the `docutils` into scope while building...

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/version-management/git-crecord/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  version = "20161216.0";
+
+  src = fetchFromGitHub {
+    owner = "andrewshadura";
+    repo = "git-crecord";
+    rev = version;
+    sha256 = "0v3y90zi43myyi4k7q3892dcrbyi9dn2q6xgk12nw9db9zil269i";
+  };
+
+  buildInput = with pythonPackages; [ docutils ];
```
Comment (Mic92) [inline pkgs/applications/version-management/git-crecord/default.nix]: `propagatedBuildInputs` is usually used for python.

Comment (matthiasbeyer) [inline pkgs/applications/version-management/git-crecord/default.nix]: I tried that as well, but it won't work. I guess `buildInputs` would suffice here, as `docutils` is only a built-time dependency, as far as I can see...

File: pkgs/applications/version-management/git-crecord/default.nix [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  version = "20161216.0";
+
+  src = fetchFromGitHub {
+    owner = "andrewshadura";
+    repo = "git-crecord";
+    rev = version;
+    sha256 = "0v3y90zi43myyi4k7q3892dcrbyi9dn2q6xgk12nw9db9zil269i";
+  };
+
+  propagatedBuildInputs = with pythonPackages; [ docutils ];
```
Comment (Mic92) [inline pkgs/applications/version-management/git-crecord/default.nix]: the following built on my machine

Comment (matthiasbeyer) [inline pkgs/applications/version-management/git-crecord/default.nix]: Hm. Maybe I made some typo somewhere and misread the error message, sorry! Thank you very much for your effort!

**Pull request merged**